### DISCUSSION TITLE
Question about list view

### DISCUSSION THREAD

**Discussion by danielkeho** [TheCadien/SuluNewsBundle#37] [Q&A] [answered]
How list view of news is created with this bundle please

**Comment by TheCadien** [comment, marked as answer]
Sorry for late response.

To get a list of news on your website you can use the SmartContent component. 
This works basically as described in the Sulu Docs. But here is an example.

Add such a block to your Template.xml.
`        
<property name="news" type="smart_content">
            <meta>
                <title lang="en">News</title>
            </meta>

            <params>
                <param name="provider" value="news"/>
                <param name="max_per_page" value="5"/>
                <param name="page_parameter" value="p"/>
      </params>
</property>
`

You will get an array with news. 
which you can go through with a For loop in twig. 
You can also find an example of a twig integration in the [docs](https://github.com/TheCadien/SuluNewsBundle/blob/master/Resources/docs/template.md)
